01-introduction.Rmd

```diff
@@ -5,7 +5,7 @@ Suggested titles were *Geography with R* and *R for GIS*, each of which has adva
 The former conveys the message that it's not just about spatial data: 
 non-spatial attribute data are inevitably interwoven with geometry data, and Geography is about more than where something is on the map.
 The latter option communicates that this is a book about using R as a *Geographic Information System* (GIS), to perform spatial operations on *geographic data* [@bivand_applied_2013].
-However, the term GIS has some negative connotations (see Table \@ref(tab:gdsl)) and fails to communicate one of R's greatest strengths:
+However, the term GIS has some negative connotations <!-- I see what you mean by negative connotations but I had to think immediately of qualitative GIS and critical GIS, i.e., scientists from the humanities criticising GIS technologies. Think of the so-called "GIS crisis", positivism, make-believe objectivity, etc.--> (see Table \@ref(tab:gdsl)) and fails to communicate one of R's greatest strengths:
```

Comment:
Thanks for this comment. @Nowosad maybe it's better to clarify what we mean here. Note the great Stan Openshaw was talking about a crisis in GIS back in 1991. Worth citing this: http://journals.sagepub.com/doi/abs/10.1068/a230621?journalCode=epna to be clear what we mean?